src/main/java/apoc/uuid/Uuid.java

```diff
@@ -0,0 +1,132 @@
+package apoc.uuid;
+
+import apoc.ApocConfiguration;
+import apoc.Description;
+import apoc.util.Util;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.kernel.internal.GraphDatabaseAPI;
+import org.neo4j.logging.Log;
+import org.neo4j.procedure.Context;
+import org.neo4j.procedure.Mode;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class Uuid {
+
+    @Context
+    public GraphDatabaseService db;
+
+    @Procedure(mode = Mode.WRITE)
+    @Description("CALL apoc.uuid.add(`name`, [label], {config}) | it will ad the uuid for the Label")
+    public Stream<UuidInfo> add(@Name("name") String name, @Name("labels") List<String> labels, @Name(value = "selector", defaultValue = "{}") Map<String, Object> selector) {
```

Comment:
Quando si aggiungere un trigger UUID va verificato se per le label definitite sono già presenti nodi e in questo caso va tenuto conto di un parametro di configurazione `addToAlreadyPresent` true/false (default: true) che fa l'aggiornamento del pregresso